src/react/endpoint/EndpointList.tsx

```diff
@@ -38,28 +34,35 @@ type Props = {
 
 function EndpointList({ endpoints, locations }: Props) {
   const dispatch = useDispatch();
-  const listRef = useRef(null);
   const showDelete = useSelector(
     (state: AppState) => state.uiEndpoints.showDelete,
   );
 
-  const handleDeleteApprove = () => {
-    if (!showDelete) {
-      return;
-    }
-
-    dispatch(deleteEndpoint(showDelete));
-  };
-
-  const handleDeleteCancel = () => {
-    dispatch(closeEndpointDeleteDialog());
-  };
+  const [hostnameFilter, setHostNameFilter] = useState();
+  /*
+   *   Enforcing a strict schema because the table interprets `undefined` values
+   *   as empty and shows a minus sign instead (which makes total sense), disregarding what is
+   *   specified in the custom render (Cell function)
+   *   Currently the API returns some objects without a `isBuiltin` property causing this undesired behavior
+   */
+  const strictSchemaEndpoints = useMemo(
+    () =>
+      endpoints.map((endpoint) => ({
+        ...endpoint,
+        isBuiltin: Boolean(endpoint.isBuiltin),
+      })),
+    [endpoints],
```

Comment:
Is endpoints stable across re-renders ?

Reply:
It's as stable as useSelector provides it